app/components/CodeSnippet/CodeSnippet.module.css

```diff
@@ -0,0 +1,6 @@
+.codeSnippetContainer {
+  & :global(.react-syntax-highlighter-line-number) {
+    color: var(--mantine-color-dark-4) !important;
```

Comment:
are these importants actually needed in a css module?

Reply:
Only for this global class, since this is for a component that's not from mantine... I'll double check though, as I did something to change the order of the css files imports to avoid using `!important`

Reply:
After confirming, they are needed, since this is an external component